integration-tests/api/helpers/product-category-seeder.ts

```diff
@@ -0,0 +1,47 @@
+import { ProductCategory } from "@medusajs/medusa"
+
+export async function productCategorySeeder(
```

Comment:
suggestion: rm, If we only use this one place we should just use factories instead. IIRC we are avoiding the seeder pattern as much as possible going forward, right @olivermrbl?

Reply:
+1

Reply:
+1